Composer/packages/tools/language-servers/language-generation/__tests__/LGServer.test.ts

```diff
@@ -57,6 +57,13 @@ function jsonEscape(str) {
 const content = jsonEscape(lgFile);
 
 describe('LG LSP server test', () => {
+  beforeEach(() => {
+    process.env.NODE_ENV = 'test';
+  });
+
+  afterEach(() => {
+    delete process.env.NODE_ENV;
+  });
```

Comment:
This might affect future tests that depend on a `process.env` of `development`. I'd try to save the original value and then restore it after all the tests are done.

```
let nodeEnv;
beforeAll(() => {
  nodeEnv = process.env.NODE_ENV;
  process.env.NODE_ENV = 'test';
});

afterAll(() => {
  process.env.NODE_ENV = nodeEnv;
})
```